package.json

```
@@ -43,6 +44,7 @@
     "CHANGELOG.md"
   ],
   "scripts": {
+    "debug": "http-server --port 3000 -c-1",
```

I think we can fall back to the default port number here.
```suggestion
    "debug": "http-server -c-1",
```